### PR TITLE
Follow-on to PR # 4581

### PR DESCRIPTION
**Pull request by mppf**
These changes address Greg's feedback on the multilocale documentation updates.

Reviewed by @gbtitus - thanks!

### CODE REVIEW

**Comment by mppf** [comment]
@gbtitus - thanks for finding the errors in these. Could you have a look at the new paragraph about tradeoffs for segments? The other changes are trivial. Thanks!

File: doc/release/multilocale.rst [hunk 1]
```diff
@@ -17,15 +17,15 @@ This document outlines the steps to get started with multi-locale Chapel.
 
 #. Configure the Chapel runtime appropriately. For most configurations, set:
 
-  .. code-block:: bash
+   .. code-block:: bash
 
      export CHPL_COMM=gasnet
 
-  Then, consider changing several these variables:
+   Then, consider changing several these variables:
```
Comment (gbtitus) [inline doc/release/multilocale.rst]: I'm sorry, I missed this one before: there's a missing "of" between "these" and "variables".

**Comment by gbtitus** [comment]
That looks good, modulo the missing "of".

File: doc/release/multilocale.rst (hunk 1 above)
Comment (mppf) [inline doc/release/multilocale.rst]: Looks like I need a compiler for English :)

Comment (mppf) [inline doc/release/multilocale.rst]: I'm going to change it to "Then, consider changing these variables" while there. I don't think "several of" adds anything.

**Pull request merged**